sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereStatement.cs

```diff
@@ -31,13 +31,22 @@ public WhereLogic Where()
         /// <inheritdoc/>
         public override AdtQueryBuilder Build()
         {
-            throw new InvalidOperationException("Invalid query: Missing WHERE logic.");
+            return _parent;
         }
 
         /// <inheritdoc/>
         public override string GetQueryText()
         {
-            return QueryConstants.Where;
+            string whereLogicString = _upstreamWhereLogic.GetQueryText();
+
+            if (whereLogicString.Length > 0)
+            {
+                return $"{QueryConstants.Where} " + whereLogicString;
```

Comment:
just use the interpolation to include the `whereLogicstring` as well.
`return $"{Queryconstants.Where} {whereLogicString}";